apps/judicial-system/web/src/routes/Court/InvestigationRequest/CourtRecord/CourtRecordForm.tsx

```diff
@@ -0,0 +1,376 @@
+              }
+              onChange={(date: Date | undefined, valid: boolean) => {
+                newSetAndSendDateToServer(
+                  'courtStartDate',
+                  date,
+                  valid,
+                  workingCase,
+                  setWorkingCase,
+                  setCourtRecordStartDateIsValid,
+                  updateCase,
+                )
+              }}
+              required
+            />
+          </Box>
+          <Box marginBottom={3}>
+            <Input
+              data-testid="courtAttendees"
+              name="courtAttendees"
+              label="Viðstaddir og hlutverk þeirra"
+              defaultValue={workingCase.courtAttendees}
+              placeholder="Skrifa hér..."
+              onChange={(event) =>
+                removeTabsValidateAndSet(
+                  'courtAttendees',
+                  event,
+                  ['empty'],
+                  workingCase,
+                  setWorkingCase,
+                  courtAttendeesEM,
+                  setCourtAttendeesEM,
+                )
+              }
+              onBlur={(event) =>
+                validateAndSendToServer(
+                  'courtAttendees',
+                  event.target.value,
+                  ['empty'],
+                  workingCase,
+                  updateCase,
+                  setCourtAttendeesEM,
+                )
+              }
+              errorMessage={courtAttendeesEM}
+              hasError={courtAttendeesEM !== ''}
+              textarea
+              rows={7}
+              required
+            />
+          </Box>
+          <Input
+            data-testid="prosecutorDemands"
+            name="prosecutorDemands"
+            label="Krafa"
+            defaultValue={workingCase.prosecutorDemands}
+            placeholder="Hvað hafði ákæruvaldið að segja?"
+            onChange={(event) =>
+              removeTabsValidateAndSet(
+                'prosecutorDemands',
+                event,
+                ['empty'],
+                workingCase,
+                setWorkingCase,
+                prosecutorDemandsEM,
+                setProsecutorDemandsEM,
+              )
+            }
+            onBlur={(event) =>
+              validateAndSendToServer(
+                'prosecutorDemands',
+                event.target.value,
+                ['empty'],
+                workingCase,
+                updateCase,
+                setProsecutorDemandsEM,
+              )
+            }
+            errorMessage={prosecutorDemandsEM}
+            hasError={prosecutorDemandsEM !== ''}
+            textarea
+            rows={7}
+            required
+          />
+        </Box>
+        <Box component="section" marginBottom={8}>
+          <Box marginBottom={2}>
+            <Text as="h3" variant="h3">
+              Dómskjöl
+            </Text>
+          </Box>
+          <CourtDocuments
+            title={`Krafa um ${caseTypes[workingCase.type]}`}
```

Comment:
This is not a 100% sound implementation. This works fine for some case types (custody, travel ban, phone tapping) but not with others. If the case type is f.x. AUTOPSY, this text would read "Krafa um krufning". I'll ask Guðný and Baldur about this on Slack.

Reply:
Goog catch.

Reply:
Created a task in Asana for this, we'll fix later